src/quicr_server_raw_session.cpp

```diff
@@ -0,0 +1,509 @@
+/*
+ *  quicr_server_raw_session.h
+ *
+ *  Copyright (C) 2023
+ *  Cisco Systems, Inc.
+ *  All Rights Reserved
+ *
+ *  Description:
+ *      This file implements a session layer between the client APIs and the
+ *      transport that uses raw data packets (either UDP or QUIC).
+ *
```

Comment:
please clarify what does raw data packets mean?

Reply:
Yeah, whoever wrote the code did so in such a way that they needed access to these.  It's the way the transport callback code is written.